tests/foundry/foundry-list.check.expected

```diff
@@ -1,484 +1,496 @@
-APRProof: AccountParamsTest.testDealConcrete:d50737a25b9af563feaa75d11ca1053406a57e4c2437645dfe1e3c9bb7059443
+APRProof: AccountParamsTest.testDealConcrete:aa6ebb29536834f3fc654cf9d98d91d29982b8ac015daeaf7efd6a747ba72776
```

Comment:
I've been meaning to ask, what causes the change in this hash?

Reply:
I actually don't know. Here is where the `proof_digest` is computed for a given proof in a Foundry property test suite: https://github.com/runtimeverification/evm-semantics/blob/18567e5fe70f8ff62d7204e112efe72bde4b1638/kevm-pyk/src/kevm_pyk/foundry.py#L102

And here is where the contract digest is computed (which is what's changing here): https://github.com/runtimeverification/evm-semantics/blob/18567e5fe70f8ff62d7204e112efe72bde4b1638/kevm-pyk/src/kevm_pyk/solc_to_k.py#LL170C13-L170C13

So it is basically a hash of all the Solidity compiler output. Could be many things changing. Likely we should be hashing less data as part of that digest to keep it more stable. @nwatson22 is working on something similar to that for the `setUp` methods specifically: https://github.com/runtimeverification/evm-semantics/issues/1795